boxsdk/auth/oauth2.py

```diff
@@ -294,7 +300,7 @@ def _update_current_tokens(self, access_token, refresh_token):
         """
         self._access_token, self._refresh_token = access_token, refresh_token
 
-    def _send_token_request_without_storing_tokens(self, data, access_token, expect_refresh_token=True):
+    def execute_token_request(self, data, access_token, expect_refresh_token=True):
```

Comment:
I think we should keep this private for now.

Not that I think it should never be public. But until we know that we definitely need this in the public API, I don't think we should clutter the API and add more backwards-compatibility restrictions for ourselves.

Reply:
sure, done